guides/framework-guides/product.mdx

```diff
@@ -15,13 +15,13 @@ Let’s see how product teams can leverage it:
 This collaborative approach between the Engineering and Product teams allows both teams to focus on their strengths, resulting in a more efficient and effective notification strategy, without constant interruptions in each other’s flow.
 
 ## Customising the workflow
-To demonstrate the customizability, we’ll assume that the Engineering team has successfully delivered the basic setup as outlined in [one of our guides.](https://docs.novu.co/guides/echo-guides/echo-react-email)
+To demonstrate the customizability, we’ll assume that the Engineering team has successfully delivered the basic setup as outlined in [one of our guides.](https://docs.novu.co/guides/framework-guides/echo-react-email)
 
 1. To start things off, you only need to visit the workflow section in our [Web Dashboard](https://web.novu.co/).
 <Frame caption="The workflow section in the Novu Web Dashboard"><img src="/images/product-team/1.png" /> </Frame>
 - You’ll see the blue lightning bolt icon right next to the workflow name symbolising that it is a code-first workflow.
 - Now, simply click on the workflow to open it and make your desired changes.
-- The green `Echo` indicator on the top right corner means that your app is successfully connected with Novu.
+- The green `Sync` indicator on the top right corner means that your app is successfully connected with Novu.
```

Comment:
I'm not sure what the "Echo" indicator is, I didn't notice it in the mockup files in figma

Reply:
@denis-kralj-novu this is in the current Dev Studio. The one we currently use to run it.
![CleanShot 2024-06-19 at 11 10 24@2x](https://github.com/novuhq/docs/assets/2946769/8dcb6654-d9cc-4796-8d40-d04789d45af7)

Reply:
Thanks for the explanation :)